### ISSUE TITLE
Parent controlled layout

### ISSUE THREAD

**Issue by souporserious** [modulz/prototype-feedback#3]
Right now Text/Heading can set their own spacing (padding/margin), should this only be a responsibility of Box and possibly other future layout components like Grid and Flex to help make composition easier?

**Comment by StephenHaney** [comment]
Great topic... I think that makes a lot of sense. Please keep these thoughts coming!

**Comment by naher94** [comment]
@souporserious that's interesting, but I guess if the goal is the mirror `HTML` and `CSS` then it makes sense for text to carry padding and margin.... @StephenHaney Question for you would be is the idea to expose the underlying technology's philosophy to the end user or hid it away? I have thoughts on this as tools designer myself but I'll hold it to see what Modulz direction is

**Comment by StephenHaney** [comment]
There's definitely a balance to find. We have an opinion but I think it will be driven by user feedback too.

In a perfect world, I'd like things to be approachable for non-coders, but not lose any of the power of html/css if you want to drop down and dial things in.

What were you thinking @naher94?

**Comment by naher94** [comment]
I would agree it should definitely be approachable for non-devs and only then you can start to uncover or share the more technical/underlying technology with the user. You could enhance the clarity of the item for those more "power user" types with some kind of hover over state that share a bit more about what a `<div>` is for example. Something like this might be nice. Mock up for illustrative purposes of course.

Related though:
I think the term `Box` is a bit confusing since it could be read as `rectangle` or `div` and in this instance it's a `div` from what I see. I wonder if `Container` would be a better term for both dev and non-dev minds alike, following the need to make the tool accessible but also showcasing the power of html/css.

![Frame 1](https://user-images.githubusercontent.com/7800629/103017585-d2e64900-4511-11eb-9f4a-a9d6c21b367d.png)
